src/main/redux/middleware/persistence.ts

```diff
@@ -44,6 +44,7 @@ export const reduxPersistMiddleware: Middleware
                     settings: prevState.settings,
                     creator: prevState.creator,
                     annotationImportQueue: prevState.annotationImportQueue,
+                    apiKeys: prevState.apiKeys,
```

Comment:
we should not persist apiKeys to the redux state but instead in a separate encrypted json file, I think